yarn.lock

```diff
@@ -7351,6 +7351,11 @@ nanoid@^4.0.0:
   resolved "https://registry.yarnpkg.com/nanoid/-/nanoid-4.0.0.tgz#6e144dee117609232c3f415c34b0e550e64999a5"
   integrity sha512-IgBP8piMxe/gf73RTQx7hmnhwz0aaEXYakvqZyE302IXW3HyVNhdNGC+O2MwMAVhLEnvXlvKtGbtJf6wvHihCg==
 
+natural-compare@^1.4.0:
```

Comment:
This wasn't added in the `package.json`, should it be?

Reply:
I think it would be okay if it was left out. It might have been added by a package I installed and then removed, but not entirely sure.